README.md

```diff
@@ -452,9 +452,9 @@ $19 = 0x5
 See [handle_test.cc](./test/handle_test.cc) for an example.
 
 ### HandleScope
-Contains a number of Local/Handle's (think pointers to objects but is managed
-by V8) and will take care of deleting the Local/Handles for us. HandleScopes
-are stack allocated
+Contains a number of local handles (like pointers to objects but are managed
```

Comment:
In [v8's source code](https://github.com/v8/v8/blob/lkgr/src/handles/handles.h#L188), they use local handles rather than Local/Handles.

Reply:
I think `Local` is essentially equivalent to `Handle`. The difference of them is `Local` is an outer api  and `Handle` is an inner api. 

So we can simplify the doc by omitting the `Local`.